packages/services/api/src/modules/target/providers/target-manager.ts

```diff
@@ -133,6 +149,20 @@ export class TargetManager {
       },
     });
 
+    const currentUser = await this.session.getViewer();
+    await this.auditLog.record({
+      eventType: 'TARGET_DELETED',
+      targetId: target,
+      targetSlug: deletedTarget.slug,
+      projectId: project,
+      metadata: {
+        organizationId: organization,
```

Comment:
TODO @TuvalSimha we never need to store the organization id as metadata in clickhouse as audit logs always belong to an organization (it would be redundant with the organization_id column).